patches/access_unpublished/DP-25897_change_unpublish_preview_links_v2.patch

```diff
@@ -0,0 +1,78 @@
+diff --git a/access_unpublished.services.yml b/access_unpublished.services.yml
```

Comment:
@arthurbaghdas This seems like a service we should be overriding rather than something we should patch. Is there a reason for going this direction?

Reply:
I guess the only reason is to to keep track of the changes in the future with the new version updates.